blei-lab/edward
# Feature/example mixture gaussian

#### Summary:

Issues: makes progress for #7, #22, #57

This adds an example of mixture modeling on synthetic data, using both mean-field variational inference and maximum a posteriori. For mean-field variational inference, the variational family is a product of Dirichlet, Gaussian, and Inverse Gamma densities. 

This also makes progress on how to build an interface allowing mean-field families that are a product of different distributions rather than only one distribution.

#### Intended Effect:

Mixture model examples are available.

#### How to Verify:

Run the new examples `mixture_gaussian.py` and `mixture_gaussian_map.py`. See that they work.

#### Side Effects:

N/A

#### Documentation:

Doc is added in the model examples.

#### Reviewer Suggestions: 

@diengadji
